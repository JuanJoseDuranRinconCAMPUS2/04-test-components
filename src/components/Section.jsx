import React from "react";

export default function Section(){
    return(
        <>
            <div>
                <div id="introSeccion">
                    <div id="seccion-img">
                        <img src="https://cdn.pixabay.com/photo/2017/07/03/20/17/colorful-2468874_640.jpg" alt="hola" />
                    </div>
                    <div id="seccion-p">
                        <p>
                        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Iure nesciunt modi excepturi molestiae explicabo quaerat iste reprehenderit saepe fugit ab incidunt, expedita illo voluptatem, ipsum enim neque optio similique porro.
                        </p>
                    </div>
                </div>
            </div>
        </>
    )
}