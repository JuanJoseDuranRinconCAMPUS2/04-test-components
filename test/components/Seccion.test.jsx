import { render } from "@testing-library/react";
import Section from "../../src/components/Section";

describe("Captura del componente <Seccion.jsx>", ()=>{
    test('Fotito :3', () => { 
        let {container} = render(<Section/>);
        expect(container).toMatchSnapshot();
    })
})
