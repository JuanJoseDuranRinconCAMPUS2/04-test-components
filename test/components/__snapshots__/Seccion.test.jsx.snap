// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captura del componente <Seccion.jsx> Fotito :3 1`] = `
<div>
  <div>
    <div
      id="introSeccion"
    >
      <div
        id="seccion-img"
      >
        <img
          alt="hola"
          src="https://cdn.pixabay.com/photo/2017/07/03/20/17/colorful-2468874_640.jpg"
        />
      </div>
      <div
        id="seccion-p"
      >
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Iure nesciunt modi excepturi molestiae explicabo quaerat iste reprehenderit saepe fugit ab incidunt, expedita illo voluptatem, ipsum enim neque optio similique porro.
        </p>
      </div>
    </div>
  </div>
</div>
`;
